visualization_graph.py

```diff
@@ -0,0 +1,65 @@
+import plotly.express as px
+from visualization import KnowledgeGraphVisualizer
+
+class Graph():
```

Comment:
Splitting up the treemap code and the lists (graph) generation is good but might need a more indicative name than just `Graph`